lib/components/viewers/stop-viewer.js

```diff
@@ -156,11 +157,23 @@ class StopViewer extends Component {
     zoomToPlace(map, stopData)
   }
 
-  _renderHeader = () => {
-    const { enableFavoriteStops, hideBackButton, showUserSettings, stopData } =
-      this.props
+  _renderHeader = (agencyCount) => {
+    const {
+      enableFavoriteStops,
+      hideBackButton,
+      showUserSettings,
+      stopData,
+      transitOperators
+    } = this.props
+
+    // We can use the first route, as this operator will only be used if there is only one operator
+    const stationOperator = transitOperators.find(
+      (o) =>
+        o.agencyId === stopData?.agencyId || stopData?.routes?.[0]?.agencyId
+    )
+
     return (
-      <div className="stop-viewer-header">
+      <div className="stop-viewer-header base-color-bg">
```

Comment:
The font color should be set where the ambient background color is set (multiple instances).

Reply:
I tried to do this, but there were too many instances of the wrong elements being colored.